src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.handler.impl.BalanceHandler;
+import core.basesyntax.strategy.handler.impl.PurchaseHandler;
+import core.basesyntax.strategy.handler.impl.ReturnHandler;
+import core.basesyntax.strategy.handler.impl.SupplyHandler;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitServiceImplTest {
+    private FruitService logger;
+    private OperationStrategy operationStrategy;
+
+    @BeforeEach
+    void setUp() {
+        logger = new FruitServiceImpl();

```

Comment:
1. Why `FruitServiceImple` instance named as `logger`?
2. Move  initialization of `FruitServiceImple` to the method @BeforeAll beforeAll(), it is not changed in different tests.
Check similar cases in other classes and fix where needed.